src/PhpSpec/Console/Command/DescribeCommand.php

```diff
@@ -64,9 +67,37 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $container = $this->getApplication()->getContainer();
         $container->configure();
 
-        $classname = $input->getArgument('class');
-        $resource  = $container->get('locator.resource_manager')->createResource($classname);
+        if ($input->getArgument('class')) {
+            $classname = $input->getArgument('class');
+        } else {
+            $questionHelper = $this->getApplication()->getHelperSet()->get('question');
+            $question = new Question('<info>Enter class to describe: </info>');
```

Comment:
One thing that I'm worried about is the way of interaction with `Symfony\Component\Console\Question\Question`.

There is `\PhpSpec\Console\ConsoleIO` class that implements work with console input/output.
And there is also `\PhpSpec\Console\Prompter` interface.
Maybe this asking stuff should be moved there too.

This will allow us to reuse autocompletion functionality in other commands, and it will make possible to write behat scenario to check autocomplete values.

Reply:
This should definitely go through the Prompter 👍

Reply:
I don't think we should reflect the existence of the symfony Question in our own interface - we can keep the prompter interface simple, I like @avant1's suggestion